client/views/static/principles/principles.html

```diff
@@ -0,0 +1,54 @@
+<template name="principles">
+	<div class="container principles">
+		<h1>Openki principles</h1>
+
+<h3> Education should be accessible to everyone</h3>
+<div>
+Openki courses should have only indicative (not fixed) prices and always accept free admissions.
+
+Mentors can still discuss with the course participants about a minimum amount expected for them to invest the time and effort required for a specialized course to be offered and/or fixed material costs (rental for tools, room, etc.)
+</div>
+
+
+<h3> Education should be a collective participatory process</h3>
+<div>
+
+Openki-courses are designed as open and collective learning processes that can be initiated and organized by anyone. In practical terms, a course is a flexible digital space that facilitates the bottom-up appropriation of different roles (mentor, host, organizer, participant). Trust building between the interested course members is expected to take place offline before or during the different related events, regular or irregular.
+
+The software itself is under continuous development according to the needs of the different groups and our vibrant github repository, http://github/Openki, provides the means for discussions and deliberation around key functionalities and features.
+</div>
+
+
+<h3>Digital platforms should respect the privacy of the users</h3>
+
+<div>
+The Openki platform allows for practically anonymous participation and does not collect any form of private data except from the really minimum to allow for reliable communication in the platform.
```

Comment:
We do analytics. Mabe we have to be more clear here. We don't sell or give away or whatever. ask a lawyer first ;)